powershell-adapter/psDscAdapter/psDscAdapter.psm1

```diff
@@ -591,4 +577,4 @@ class DscResourceInfo {
     [string] $ImplementedAs
     [string] $CompanyName
     [System.Collections.Generic.List[DscResourcePropertyInfo]] $Properties
-}
+}
```

Comment:
Best practice is to have a newline at the end as some tools read until a newline